libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -112,12 +116,35 @@ const AP_Param::GroupInfo AP_OSD::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("_H_OFFSET", 11, AP_OSD, h_offset, 32),
 
+#if OSD_CRSF_PANELS_ENABLED
+    // @Param: _W_RSSI
+    // @DisplayName: RSSI warn level (in %)
+    // @Description: Set level at which RSSI item will flash (-dBm)
+    // @Range: -128 0
+    // @User: Standard
+    AP_GROUPINFO("_W_RSSI", 12, AP_OSD, warn_rssi, -100),
```

Comment:
RSSI is a standard metric not limited to CRSF, your PR should allow this feature to be used elsewhere

Reply:
I think you have 11 characters free here (so OSDx_ + addition), so you should try and make the name more meaningful. _WARN_RSSI would fit for instance

Reply:
Rather than change the whole parameter I think just change the default in a #define somewhere which is what you are looking for